Unable to load RobotModel

I tried to run this code on ubuntu 20.04.5 with ros noetic and gazebo 11.11 .
All robot nodes in RViz are error and show "No transform from [panda_hand] to [world]".

I got the following error :

```
[Err] [Model.cc:123] Error Code 23 Msg: FrameAttachedToGraph error, Non-LINK vertex with name [__model__] is disconnected; it should have 1 outgoing edge in MODEL attached_to graph.
[Err] [Model.cc:123] Error Code 23 Msg: Graph with __model__ scope has sink vertex named [__model__] when starting from vertex with name [__model__], which is not permitted.
```

and

```
[ERROR] [1668529954.882822426, 22554.300000000]: Action client not connected: effort_joint_trajectory_controller/follow_joint_trajectory
```

--- a/launch/includes/spawn_mobile_manipulator.launch
+++ b/launch/includes/spawn_mobile_manipulator.launch
@@ -62,6 +62,7 @@
 
             <!-- MoveIt! Main -->
             <include file="$(find panda_moveit_config)/launch/move_group.launch">
+                  <arg name="load_robot_description" value="false"/>
                   <arg name="info" value="true"/>
                   <arg name="debug" value="false"/>
             </include>

